### ISSUE TITLE
Containers get an internet connection through VPN but are inaccessible via mapped ports

### ISSUE THREAD

**Issue by providentitsolutions** [bubuntux/nordvpn#306]
**Describe the bug**
The NordVPN container connects to the VPN as expected and can communicate outbound, containers routed through it via --net/network_mode can also connect to the internet via the VPN (tested with "curl ipinfo.io") but are inaccessible via a web browser through the mapped ports. If I remove --net/network_mode, or change the container to route through qmcgaw/gluetun it works as expected.

**To Reproduce using docker CLI**
docker run -ti -p 8080:80 --cap-add=NET_ADMIN --name vpn -e USER=user@name.com -e PASS='nordpassword' -e TECHNOLOGY=NordLynx -d ghcr.io/bubuntux/nordvpn:latest
docker run -t --net=container:vpn -d --name nccoffee thenetworkchuck/nccoffee:frenchpress

**To Reproduce using docker-compose**
docker-compose.yml if used  (hide credentials)
```
  nordvpn:
    image: ghcr.io/bubuntux/nordvpn:latest
    cap_add:
      - NET_ADMIN # Required for VPN Networking
    container_name: nordvpn
    environment:
      - USER=user@name.com
      - PASS=nordvpnpassword
      - TECHNOLOGY=NordLynx
      - TZ=Europe/London
    ports:
      - 8080:80/tcp # Network Chuck testing webserver
      - 8888:8888/tcp # qBitTorrent Web UI
    sysctls:
      - net.ipv6.conf.all.disable_ipv6=1 # Disable IPv6
    restart: always

  nccoffee:
    image: thenetworkchuck/nccoffee:frenchpress
    container_name: nccoffee
    network_mode: "service:nordvpn"
    depends_on:
      - nordvpn
```

**Expected behavior**
nccoffee should be reachable via http://dockerhostip:80 but instead the connection is refused (same with qbittorrent, radarr, sonarr etc. on various different ports). If I swap bubuntux/nordvpn with qmcgaw/gluetun it works. If I remove --net/network_mode and add the port mappings directly to the other containers, that also works.

**Logs**
docker logs --follow nordvpn = Connected, no errors
docker logs --follow <child_container> = As expected, no errors
(docker exec -it <child_container> bash) curl ipinfo.io = VPN ISP information, as expected
The only thing that strikes me as odd is I have the ENV variable to disable IPv6 in my compose file but when I view port mappings IPv6 is enabled:
docker port nordvpn = 
8080/tcp -> 0.0.0.0:8080
8080/tcp -> :::8080

**Additional context**
Ubuntu Server 20.04.3 LTS (fresh install, just docker & docker compose installed)
docker --version = Docker version 20.10.12, build e91ed57
docker-compose --version = docker-compose version 1.25.0, build unknown
uname -a = Linux dockerserver 5.4.0-92-generic #103-Ubuntu SMP Fri Nov 26 16:13:00 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

**Comment by providentitsolutions** [comment]
After running nicolaka/netshoot and doing some troubleshooting I found that I can access the Web UI from other docker containers using network_mode:nordvpn (curl 127.0.0.1:8888 displayed qBittorrent's index.html source) but still nothing from any devices on my LAN or the docker host machine itself. I had the same result with the nordvpn containers using the default bridge network or a custom bridge network.

After a LOT more testing I added the environment variable "NETWORK=0.0.0.0/0" and hey presto it now works from my LAN. 

I'm leaving this open as I couldn't spot the reason behind this in the documentation so it could still be a bug, I'd be really interested to hear your thoughts as I'm still new to docker.
